x-pack/plugins/index_lifecycle_management/public/application/services/policies/policy_validation.ts

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { validateHotPhase } from './hot_phase';
+import { validateWarmPhase } from './warm_phase';
+import { validateColdPhase } from './cold_phase';
+import { validateDeletePhase } from './delete_phase';
+import { ColdPhase, DeletePhase, HotPhase, Phase, Policy, PolicyFromES, WarmPhase } from './types';
+
+export const numberRequiredMessage = i18n.translate(
+  'xpack.indexLifecycleMgmt.editPolicy.numberRequiredError',
+  {
+    defaultMessage: 'A number is required.',
+  }
+);
+
+// TODO validation includes 0 -> should be non-negative number?
```

Comment:
Is this comment still relevant?

Reply:
This is something I noticed and want to check with es-docs team at some point. We have a check for `>= 0` but the wording is _Only positive numbers are allowed._ and there is also `>0` check with wording _Only numbers above 0 are allowed._. So it looks like the first wording is not correct and needs an update hence the comment.

Reply:
Cool. At the end ES will make the final validation so I don't think it is a big problem.